### PR TITLE
added the dependency diagram

### PR DESCRIPTION
**Pull request by theRealImy**
Please review it it is OK to add the dependency diagram to the readme.
solidos

### CODE REVIEW

File: documentation/solidos_dependencies.svg [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+<g id="edge8" class="edge">
+<title>app&#45;&gt;mashlib</title>
+<path fill="none" stroke="#f781bf" d="M167.8,-431.7C167.8,-423.98 167.8,-414.71 167.8,-406.11"/>
+<polygon fill="#f781bf" stroke="#f781bf" points="171.3,-406.1 167.8,-396.1 164.3,-406.1 171.3,-406.1"/>
+</g>
+<!-- native -->
+<g id="node10" class="node">
+<title>native</title>
+<ellipse fill="none" stroke="#f781bf" cx="303.8" cy="-450" rx="61.99" ry="18"/>
+<text text-anchor="middle" x="303.8" y="-446.3" font-family="Times,serif" font-size="14.00" fill="#f781bf">Native Aps</text>
```
Comment (bourgeoa) [inline documentation/solidos_dependencies.svg:172]:
```suggestion
<text text-anchor="middle" x="303.8" y="-446.3" font-family="Times,serif" font-size="14.00" fill="#f781bf">Native Apps</text>
```

**Comment by timbl** [comment]
Looks good to me.  For some reason the actual review form I can't get up, but happy with this PR

**Review by timbl — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -128,6 +128,9 @@ It also makes use of —
 Let's take a look at an architecture diagram of SolidOS:
 ![SolidOS architectural overview](documentation/architecture.svg)
 
+A colorful dependency tree can also be seen next:
```
Comment (TallTed) [inline README.md:131]:
```suggestion
A colorful dependency tree can be seen here:
```

**Pull request merged**